fiftyone/core/dataset.py

```diff
@@ -4244,12 +4244,22 @@ def _clear(self, view=None, sample_ids=None):
         else:
             contains_videos = self._contains_videos(any_slice=True)
 
+        ops = []
         if sample_ids is not None:
-            d = {"_id": {"$in": [ObjectId(_id) for _id in sample_ids]}}
+            batch_size = fou.recommend_batch_size_for_value(
+                ObjectId(), max_size=100000
```

Comment:
should we define this as a constant somewhere?

Reply:
It's actually not a constant, as `recommend_batch_size_for_value()` decides what batch size to use based on the current `fo.config` which could be changed dynamically by the user:
https://github.com/voxel51/fiftyone/blob/c699f8dfc16acd01d17acc012aa961e691e538e8/fiftyone/core/utils.py#L1678

Reply:
I think he meant the `max_size=100000`